flocker/provision/_install.py

```diff
@@ -11,12 +11,27 @@
 from urlparse import urljoin
 from characteristic import attributes
 
+from flocker import __version__ as flocker_version
+from flocker.docs import get_doc_version, is_release
 from ._common import PackageSource
 
 ZFS_REPO = ("https://s3.amazonaws.com/archive.zfsonlinux.org/"
             "fedora/zfs-release$(rpm -E %dist).noarch.rpm")
-CLUSTERHQ_REPO = ("https://storage.googleapis.com/archive.clusterhq.com/"
-                  "fedora/clusterhq-release$(rpm -E %dist).noarch.rpm")
+
+if is_release(get_doc_version(flocker_version)):
+    key = 'marketing'
+else:
+    key = 'development'
```

Comment:
To clarify: Do you mean that we should not separate marketing and testing RPMs in S3? If so, would you prefer that I remove this feature?